samples/ClassroomLabs/Scripts/EthicalHacking/Create-EthicalHacking-LabsAccount.ps1

```diff
@@ -0,0 +1,111 @@
+    [string]$Email,
+
+    [parameter(Mandatory = $false, HelpMessage = "Default username for all VMs")]
+    [string]$Username = "AdminUser",
+
+    [parameter(Mandatory = $false, HelpMessage = "Default password for all VMs")]
+    [string]$Password = "P@ssword1!",
+
+    [parameter(Mandatory = $false, HelpMessage = "Default location for lab account")]
+    [string]$Location = "centralus",
+
+    [parameter(Mandatory = $false, HelpMessage = "Default location for lab account")]
+    [string]$ClassName = "EthicalHacking"
+)
+
+###################################################################################################
+#
+# Handle all errors in this script.
+#
+
+trap {
+    # NOTE: This trap will handle all errors. There should be no need to use a catch below in this
+    #       script, unless you want to ignore a specific error.
+    $message = $Error[0].Exception.Message
+    if ($message) {
+        Write-Host -Object "`nERROR: $message" -ForegroundColor Red
+    }
+
+    Write-Host "`nThe script failed to run.`n"
+
+    # IMPORTANT NOTE: Throwing a terminating error (using $ErrorActionPreference = "Stop") still
+    # returns exit code zero from the PowerShell script when using -File. The workaround is to
+    # NOT use -File when calling this script and leverage the try-catch-finally block and return
+    # a non-zero exit code from the catch block.
+    exit -1
+}
+
+###################################################################################################
+#
+# Main execution block.
+#
+
+# Install AzLab module
+Import-Module ..\..\Modules\Library\Az.LabServices.psm1 -Force
+
+# Configure parameter names
+$rgName     = "$($ClassName)RG_" + (Get-Random)
+$labAcctName     = "$($ClassName)Acct_" + (Get-Random)
+$labName    =  "$($ClassName)Lab"
+
+# Create resource group
+Write-Host "Creating resource group $rgName"
+$rg = New-AzResourceGroup -Name $rgName -Location $Location
+    
+# Create Lab Account
+Write-Host "Creating lab account $labAcctName"
+$labAcct  = New-AzLabAccount -ResourceGroupName $rgName -LabAccountName $labAcctName
+
+# Ensure that image needed for the VM is available
+$imageName = "Windows Server 2019 Datacenter"
+Write-Host "Locating '$imageName' image for use in template virtual machine"
+$imageObject = $labAcct | Get-AzLabAccountGalleryImage | Where-Object -Property Name -EQ $imageName
+  
+# Only create lab if image type is found
+if($imageObject) {
+
+    # Create lab on the lab account
+    Write-Host "Creating $labName with '$($imageObject.Name)' image"
+    Write-Host "  Warning: Creating template vm may take up to 20 minutes." -ForegroundColor 'Yellow'
+    $lab = $labAcct | New-AzLab -LabName $labName -UserName $Username -Password $Password -SharedPasswordEnabled -UsageQuotaInHours 10 -Size "Virtualization" -Image $imageObject
+    Write-Host "Lab has been created. Credentials for VM template are '$Username' for the username and '$Password' for the password."
```

Comment:
I've removed this from the script.  Rationale for putting it in there was because if someone didn't supply credentials as parameters they might not know what the credentials are.